Modules/Core/Common/include/itkMacro.h

```diff
@@ -1098,14 +1097,11 @@ compilers.
   virtual void Set##name(type * _arg)                  \
   {                                                    \
     itkDebugMacro("setting " << #name " to " << _arg); \
-    CLANG_PRAGMA_PUSH                                  \
-    CLANG_SUPPRESS_Wfloat_equal                        \
     if (this->m_##name != _arg)                        \
```

Comment:
If `type` is float, this will be a floating point comparison. And this macro is used by remote modules and other external code, not just by ITK proper.

Reply:
> If `type` is float, this will be a floating point comparison.

@dzenanz Thanks, but please double-check!  The argument `_arg` is declared as `type *`, so if `type` is float, then `_arg` is a _float pointer_, not a float.  The comparison `this->m_##name != _arg` _always_ does pointer comparison, with this specific macro.

Reply:
I indeed failed to notice is it a float pointer 😄